delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/marshalling/RdfMarshalling.scala

```diff
@@ -65,11 +65,19 @@ trait RdfMarshalling {
   /**
     * NTriples -> HttpEntity
     */
-  implicit val ntriplesMarshaller: ToEntityMarshaller[NTriples] =
+  implicit val nTriplesMarshaller: ToEntityMarshaller[NTriples] =
     Marshaller.withFixedContentType(ContentType(`application/n-triples`)) { case NTriples(value, _) =>
       HttpEntity(`application/n-triples`, ByteString(value))
     }
 
+  /**
+    * NQuads -> HttpEntity
+    */
+  implicit val nQuadsMarshaller: ToEntityMarshaller[NQuads] =
```

Comment:
missing tests for this

Reply:
No. This is tested in `RdfMarshallingSpec`

Reply:
Right, I somehow missed them.